### ISSUE TITLE
Prediction shape from compression model

### ISSUE THREAD

**Issue by kartikp7** [openvinotoolkit/nncf#998]
Hi,

I am new to nncf and trying to use the Tensorflow version to simply compress a pertained model and evaluate it. I am trying to use CIFAR10 with ResNet - pruning with geometric median (from examples/tensorflow/classification/configs) and followed the example. I am able to compress the model, however when I run the predict() function, the output shape seems to have a 2048 dimension which I am not sure what it refers to. For CIFAR 10,000 test images, I wanted to just predict and evaluate for accuracy etc. but the output shape is (10000, 1, 1, 2048), where does the 2048 come from?  This is the code I am running:
```
# Load a configuration file to specify compression
nncf_config = NNCFConfig.from_json(json_path)

# Load CIFAR 10 dataset
(train_images, train_labels), (test_images, test_labels) = tf.python.keras.datasetscifar10.load_data()
train_images, test_images = train_images / 255.0, test_images / 255.0

batch_size = 128
train_dataset = tf.data.Dataset.from_tensor_slices((tf.cast(train_images, tf.float32),tf.cast(train_labels, tf.int64)))
train_dataset = train_dataset.shuffle(50000).batch(batch_size)

model = ResNet50(include_top=False, weights='imagenet',input_shape=(32,32,3), classes=train_labels.shape[1])
nncf_config = register_default_init_args(nncf_config, train_dataset, batch_size=1)

# Apply the specified compression algorithms to the model
compression_ctrl, compressed_model = create_compressed_model(model, nncf_config)

# Save compressed model
compression_ctrl.export_model("pruned_resnet50.h5", save_format='h5')

# Get predictions on test set
predictions = compressed_model.predict(test_images)
print(predictions.shape)
```

Thank you!
